lib/Onyx.js

```diff
@@ -853,24 +874,9 @@ function set(key, value) {
         return remove(key);
     }
 
-    // eslint-disable-next-line no-use-before-define
-    if (hasPendingMergeForKey(key)) {
-        Logger.logAlert(`Onyx.set() called after Onyx.merge() for key: ${key}. It is recommended to use set() or merge() not both.`);
-    }
-
-    // If the value in the cache is the same as what we have then do not update subscribers unless they
-    // have initWithStoredValues: false then they MUST get all updates even if nothing has changed.
-    if (!cache.hasValueChanged(key, value)) {
-        cache.addToAccessedKeys(key);
```

Comment:
We are removing this, but not adding it back anywhere. Is that intentional?

Reply:
I simplified [this section](https://github.com/chrispader/react-native-onyx/blob/29cc4a31132fb7143d68f10c6b44608bc20ec659/lib/Onyx.js#L857-L861) based on [a suggestion](https://github.com/Expensify/react-native-onyx/pull/238#discussion_r1260997522) by @kidroca

Reply:
Ok, I see. so it happens when we call `cache.set()` now.